src/Playwright/Playwright.csproj

```diff
@@ -35,33 +34,21 @@
     <PackageReference Include="Microsoft.SourceLink.GitHub" Version="1.1.1" PrivateAssets="All" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" Version="6.0.0" />
   </ItemGroup>
-  <Target Name="DedupeDriver" AfterTargets="CopyFilesToOutputDirectory" BeforeTargets="CopyPlaywrightFilesToOutput">
-    <ItemGroup>
-      <DriverPackage Include=".drivers\linux\package\**" />
-      <DriverNodeLicense Include=".drivers\linux\LICENSE" />
-      <DriverNodeLinuxX64 Include=".drivers\linux\node" />
-      <DriverNodeLinuxArm64 Include=".drivers\linux-arm64\node" />
-      <DriverNodeMac Include=".drivers\mac\node" />
-      <DriverNodeWin32_x64 Include=".drivers\win32_x64\node.exe" />
-      <DriverShell Include="Scripts\playwright.sh" />
-      <DriverCmd Include="Scripts\playwright.cmd" />
-    </ItemGroup>
-    <Copy SourceFiles="@(DriverPackage)" DestinationFiles="@(DriverPackage-&gt;'.playwright\package\%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Copy SourceFiles="@(DriverNodeLicense)" DestinationFiles="@(DriverNodeLicense-&gt;'.playwright\node\%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Copy SourceFiles="@(DriverNodeLinuxX64)" DestinationFiles="@(DriverNodeLinuxX64-&gt;'.playwright\node\linux-x64\%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Copy SourceFiles="@(DriverNodeLinuxArm64)" DestinationFiles="@(DriverNodeLinuxArm64-&gt;'.playwright\node\linux-arm64\%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Copy SourceFiles="@(DriverNodeMac)" DestinationFiles="@(DriverNodeMac-&gt;'.playwright\node\mac\%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Copy SourceFiles="@(DriverNodeWin32_x64)" DestinationFiles="@(DriverNodeWin32_x64-&gt;'.playwright\node\win32_x64\%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Copy SourceFiles="@(DriverShell)" DestinationFiles="@(DriverShell-&gt;'.playwright\node\linux-x64\%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Copy SourceFiles="@(DriverShell)" DestinationFiles="@(DriverShell-&gt;'.playwright\node\linux-arm64\%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Copy SourceFiles="@(DriverShell)" DestinationFiles="@(DriverShell-&gt;'.playwright\node\mac\%(RecursiveDir)%(Filename)%(Extension)')" />
-    <Copy SourceFiles="@(DriverCmd)" DestinationFiles="@(DriverCmd-&gt;'.playwright\node\win32_x64\%(RecursiveDir)%(Filename)%(Extension)')" />
-  </Target>
   <ItemGroup>
-    <None Include=".playwright\**" Pack="true" PackagePath=".playwright" />
+    <None Remove=".drivers\**" />
+    <None Include=".drivers\linux\package\**" Link=".playwright\package\%(RecursiveDir)%(Filename)%(Extension)" CopyToOutputDirectory="PreserveNewest" Pack="true" PackagePath=".playwright\package" />
+    <None Include=".drivers\linux\LICENSE" Pack="true" PackagePath=".playwright\node" />
```

Comment:
`LICENSE` is only packed and not copied to output directory since it isn't technically needed there. Please let me know if it should also be copied.